src/dwd_dynam.jl

```diff
@@ -186,7 +221,7 @@ The readout function `r` may depend on the state, history, parameters and time,
 If it is left out, then ``r=u``.
 """
 const DelayMachine{T,I} = Machine{T, I, DelayDirectedSystem{T}}
-const InstantaneousDelayMachine{T} = Machine{T, InstantaneousDirectedInterface{T}, DelayDirectedSystem{T}}
```

Comment:
Same comment as above. Make the single-typed ports the default and have the vector interface only if explicitly specified by the user.

Reply:
done.